src/Controls/src/Core/TimeChangedEventArgs.cs

```diff
@@ -0,0 +1,22 @@
+﻿#nullable disable
+using System;
+
+namespace Microsoft.Maui.Controls
+{
+	/// <include file="../../docs/Microsoft.Maui.Controls/TimeChangedEventArgs.xml" path="Type[@FullName='Microsoft.Maui.Controls.TimeChangedEventArgs']/Docs/*" />
```

Comment:
Thanks for the docs! The new way going forward is to rather put the docs directly in the code. Could you move the docs out of the separate xml file (and delete) and into the cs files?

Reply:
I've moved them, I wasn't sure what to do with the Remarks & Value tags so I have preserved them. I also don't know if you prefer all on 1 line:
```xml
<summary>The time that the user entered.</summary>
```
or tags seperated by line:
```xml
<summary>
The time that the user entered.
</summary>
```

Reply:
Multi-line is fine, but we do a mix so either way does not matter.